src/Overrides/Container.php

```diff
@@ -708,7 +708,7 @@ public function get(string $id)
                 throw $e;
             }
 
-            throw new EntryNotFoundException($id, $e->getCode(), $e);
+            throw new EntryNotFoundException($id, is_int($e->getCode()) ? $e->getCode() : 0, $e);
```

Comment:
`getCode` - always return `int` ?

Reply:
@valeriisotnyk That was a patch applied from the framework: https://github.com/laravel/framework/pull/42393

Reply:
@michael-rubel understand. ok :)